README.md

```diff
@@ -38,10 +41,14 @@ Mark the code you want to highlight with the curl class:
 or use JavaScript to dynamically highlight text:
 
 ```javascript
-hljs.registerLanguage('curl', window.hljsDefineArcade);
+hljs.registerLanguage('curl', window.hljsDefineCurl);
```

Comment:
This shouldn't need to be mentioned now since the CDN `dist` build does this automagically.

Reply:
good catch, thought I fixed that :(